controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -234,10 +234,17 @@ function execute($request) {
 
 			$user = $request->getUser();
 
+			// Get a list of author user IDs
+			$authorUserIds = array();
+			$stageAssignmentDao = DAORegistry::getDAO('StageAssignmentDAO');
+			$submitterAssignments = $stageAssignmentDao->getBySubmissionAndRoleId($submission->getId(), ROLE_ID_AUTHOR);
+			$submitterAssignment = $submitterAssignments->next();
+			assert($submitterAssignment); // At least one author should be assigned
```

Comment:
If an editor unassigns all authors from the stage, this assert might fail... should we avoid that happening?